pyproject.toml

```diff
@@ -5,9 +5,7 @@
 line-length = 88
 
 [tool.poetry]
-authors = [
```

Comment:
Guess you should add your name here too :) -- alternatively this should maybe just be some mail for the Chameleon project 🤷🏻